.github/actions/run-e2e-leg/action.yaml

```diff
@@ -71,6 +71,10 @@ inputs:
     description: 'If the test needs vertica legacy image'
     required: false
     default: 'false'
+  need-vproxy-image:
```

Comment:
This isn't needed. We can know if it's needed by check value of vproxy-image. If it's empty, we know we don't use it in that test suite.